### ISSUE TITLE
situations not adding minDecimals

### ISSUE THREAD

**Issue by jedierikb** [EmKayDK/jstepper#4]
In this example:
```
$('#txtTesting').jStepper({minValue:0, maxValue:23, minLength:2, minDecimals:3, maxDecimals:3, decimalSeparator:'.'});
```
If I enter 24, the textfield is automatically reverted to 23 -- super!
But I expected it to revert to 23.000

Also, if I enter 2 and blur focus on the textfield, it remains 2.  I expected it to change to 02.000

**Comment by EmKayDK** [comment]
Thank you for the bug report. I will investigate and make a fix as soon as possible.

**Comment by EmKayDK** [comment]
This bug is now solved with the 1.5.0 release.

Thanks again :+1:
